index.json

```diff
@@ -2,5 +2,7 @@
   "aws/monolith",
   "aws/microservice-ecommerce",
   "aws/datalake",
-  "docker/simple-demo-app"
+  "docker/simple-demo-app",
+  "GlassFish",
+  "PetPortalServiceNowExtended"
```

Comment:
I suggest to rename this to `servicenow-petportal-demo` and move it under `integration` so it would be `integration/servicenow-petportal-demo` here